DEVELOPMENT.md

```diff
@@ -0,0 +1,39 @@
+## Run OpenDID without Docker
+
+To run the OpenDID locally,
+
+1. Run the setup or add your `config.yaml` file with your seed.
+
+```bash
+ENDPOINT=peregrine ./scripts/setup.sh "build hill second flame trigger simple rigid cabbage phrase evolve final eight"
+
+```
+
+1. Change the base path in the `config.yaml` to
+
+```
+basePath: ./login-frontend/dist
```

Comment:
This can be done via sed `sed -i 's|basePath: /srv|basePath: ./login-frontend/dist|' ./config.yaml`

Reply:
sed commands depend on version and platform, let's let people do it manually.